posthog/tasks/test/test_email.py

```diff
@@ -98,6 +98,27 @@ def test_send_fatal_plugin_error(self, MockEmailMessage: MagicMock) -> None:
         assert mocked_email_messages[0].send.call_count == 1
         assert mocked_email_messages[0].html_body
 
+    def test_send_fatal_plugin_error_with_settings(self, MockEmailMessage: MagicMock) -> None:
+        mocked_email_messages = mock_email_messages(MockEmailMessage)
+        plugin = Plugin.objects.create(organization=self.organization)
+        plugin_config = PluginConfig.objects.create(plugin=plugin, team=self.team, enabled=True, order=1)
+        self._create_user("test2@posthog.com")
+        membership = self.user.organization_memberships.get()
+        membership.send_plugin_alerts = True
+        membership.save()
+
+        send_fatal_plugin_error(plugin_config.id, "20222-01-01", error="It exploded!", is_system_error=False)
+
+        # Should only be sent to user2
+        assert len(mocked_email_messages[0].to) == 1
```

Comment:
non-blocking

Rather than asserting the length, it's not much more to assert the email address. It will be more obvious what the functionality is, and gives a stronger guarantee that we're actually emailing the expected email addresses.